lib/jazzy/grouper.rb

```diff
@@ -69,7 +69,42 @@ def self.group_custom_categories(docs, doc_index)
         children.each.with_index { |child, i| child.nav_order = i }
         make_group(children, category['name'], '')
       end
-      [group.compact, docs]
+
+      globals_grouped_by_file = self.group_globals_by_file(docs, doc_index)
+      [group.compact, docs + globals_grouped_by_file]
```

Comment:
Adding `globals_grouped_by_file` to the end of docs makes the `Constants` and `Functions` sections show up at the end of other default sections, and also prevents the other default groups from being prepended with "Other".

The order of the sections, and file level groups in them appears to be consistent to be. Is there a specific way to ensure order is maintained that you would recommend?
I could sort the files in each group, and symbols in each file alphabetically to be sure. Should I?

cc: @johnfairh

Reply:
You need to go back to `group_docs()` and figure out the code there -- the advice to start in this routine was based (again :-)) on your other design.

Ruby hash iteration order is guaranteed - I think that is all you need to prove that you don't need any further sorting.